### ISSUE TITLE
Page - Aside - TeXt Theme

### ISSUE THREAD

**Issue by kitian616** [kitian616/gitalk-demo#80]
https://tianqi.name/jekyll-TeXt-theme/page/aside.html 

 A post with aside on the right.

**Comment by YunChen011** [comment]
I found that if I have header 1 before "A post with aside on the right.", namely, the first line of the article is "## Header1", the  title property ("Page - Aside") in the front matter can't work and it would be replaced by "Header1". Is it a bug or I did something wrong?
Thanks!

**Comment by kitian616** [comment]
> @RENO216
> I found that if I have header 1 before "A post with aside on the right.", namely, the first line of the article is "## Header1", the  title property ("Page - Aside") in the front matter can't work and it would be replaced by "Header1". Is it a bug or I did something wrong?
> Thanks!

What does "the  title property ("Page - Aside") in the front matter can't work" means, the article have a wrong title?

**Comment by YunChen011** [comment]
@kitian616 Yes, the problem happened before but now it has been resolved. Did you fix it? Thanks!
